ui/test/unit/appointments/controllers/manage/calendar/appointmentsCalendarViewController.spec.js

```diff
@@ -474,5 +480,65 @@ describe('AppointmentsCalendarViewController', function () {
         scope.getAppointmentsForDate("Wed Sep 06 2017 00:00:00 GMT+0530 (IST)");
         expect(scope.providerAppointments.resources.length).toEqual(1);
         expect(scope.providerAppointments.resources[0].provider.uuid).toEqual("no-provider-uuid");
-    })
+    });
+
+    it('should get config value for autoRefreshIntervalInSeconds', function () {
+        expect(appDescriptor.getConfigValue).toHaveBeenCalledWith('autoRefreshIntervalInSeconds');
+    });
+
+    it('should call interval function when autoRefreshIntervalInSeconds is defined', function () {
+        appDescriptor.getConfigValue.and.callFake(function (value) {
+            if (value === 'autoRefreshIntervalInSeconds') {
+                return 10;
+            }
+            return undefined;
+        });
+
+        createController();
```

Comment:
@rajur7, If I am not wrong, you can pull `createController()` to `beforeEach` method

Reply:
@sumanmaity112 , we are calling createController method to initialise the private variables required for these tests.

Reply:
@sowmika148, then would it be `beforeEach` thing?